source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -127,15 +127,15 @@ def get_lon_lat(obj):
     return lon, lat
 
 
-def create_address_field(geocoding):
+def create_address_field(geocoding, poi_lat=None, poi_lon=None):
     if not geocoding:
         return None
     coord = geocoding.get('coord', {})
-    lat = str(coord.get('lat')) if coord and coord.get('lat') else None
-    lon = str(coord.get('lon')) if coord and coord.get('lon') else None
-
-    return {
-        "id": geocoding.get('id'),
+    lat = str(coord.get('lat')) if coord and coord.get('lat') else poi_lat
+    lon = str(coord.get('lon')) if coord and coord.get('lon') else poi_lon
+    address_id = '{lon};{lat}'.format(lon=lon, lat=lat)
+    resp = {
+        "id": address_id,
```

Comment:
Why the Poi address ID is now long;lat ? It doesn't exist a associated ID ?

Reply:
address ids are coords. If we don't have reverse geocoding, an empty address can be generated with such an id.

Reply:
:+1: